decentraland/unity-renderer
# fix: negative loading value in loading screen

## What does this PR change?

Adds a check to avoid negative values in the ResourcesLoadTracker progress method

## How to test the changes?


1. Go to position=84,-15
2. See that you dont get a -200& loading

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
